test/concurrent/RecycleBinTest.ooc

```diff
@@ -13,13 +13,15 @@ use concurrent
 MyClass: class {
 	content: Int
 	bin: static RecycleBin<This>
-	freed: static VectorList<This>
 	allocated: static Int = 0
+	freed: static Int = 0
```

Comment:
This way we can actually remove the created classes, otherwise they could never be freed.